### ISSUE TITLE
Unclear how this handles status updates

### ISSUE THREAD

**Issue by sukima** [atmos/hubot-deploy#52]
In the [README](https://github.com/atmos/hubot-deploy/blob/master/README.md) the image below suggests that this Hubot script responds to GitHub's `deployment_status` event. However I am unable to find any reference to any `robot.router.post`.

![](https://f.cloud.github.com/assets/38/2331137/77036ef8-a444-11e3-97f6-68dab6975eeb.jpg)

The [ChatOps](https://github.com/atmos/hubot-deploy/blob/master/docs/chatops.md) also misses an opportunity to explain the relationship between Hubot, GitHub, and your own deployment server.

Lastly the [config-file](https://github.com/atmos/hubot-deploy/blob/master/docs/config-file.md) states that the `provider` property in the `apps.json` is **required** yet again I was unable to find any code references to this. I not sure why it is required.

Ultimately, how can (or is it possible) to use this script to

1. POST a `deployment` to GitHub (**yes**).
2. Provide status updates to the chat when Hubot recieves a GitHub `deployment_status` event (**unknown**).

**Comment by atmos** [comment]
/cc #41 

I'll see if I can get this going in hubot sometime soon

**Comment by atmos** [comment]
Are you still trying to use this? I added support for these kinds of events inside of hubot this week.

**Comment by sukima** [comment]
Oh, yeah I ended up rolling my own webhook based deploy system and avoided GitHub's deployment API all together. Their API was just to undocumented and confusing to really make much sense of it. Found it was easier to K.I.S.S. the process on my own.

Still, knowing that this is now available will help when my janky deploy setup borks. Especially since pitching _an app that uses webhooks to deploy the website_ is harder to sell then an app that uses *GitHub* webhooks to deploy the website ¯\\\_(ツ)\_/¯

Still than you for putting this together.
